### ISSUE TITLE
Hero over kinematic - and question about jump!

### ISSUE THREAD

**Issue by nebular** [lo-th/phy#29]
Hello !

So i have been fiddling with Hero so he can now board platforms and move alongside them  -- the development was nearly finished  I just added a couple things here and there. Do you think it is a good solution ?

https://gist.github.com/nebular/81025fdb25ed6cc7b563437948ad3989
https://www.youtube.com/watch?v=8hva2y7IyWo

also im trying to fix the jump and always come up with everything screwed up! Why is it so difficult to add the impulse vertical :( can you share some advice about that ?

thanx in advance!

**Comment by lo-th** [comment]
yes i have not finish code, is based on great work made by Erdong Chen
https://github.com/ErdongChen-Andrew/CharacterControl
but miss many option, i will add them next and clean old code 
when work on all engine

**Comment by nebular** [comment]
oh - thanks for sharing ! it is indeed an amazing work --- it if wasnt by the rapier sync raycasting calls it could be wrapped nearly as-is-- but now I understand a lot more about the code of Hero. Great to hear you have plans to continue porting the character.I can help you cleaning code when it is ready if you want help.
